src/plugins/dashboard/public/plugin.tsx

```diff
@@ -344,7 +344,7 @@ export class DashboardPlugin
     registerDashboardProvider({
       savedObjectsType: 'dashboard',
       savedObjectsName: 'Dashboard',
-      appId: 'dashboards',
+      appId: 'dashboard',
```

Comment:
This does seem better - was it necessary as part of this fix?

Reply:
yea it seems to me that this is not part of the fix. Is there an underlying reason to change the `appId`?

Reply:
This was necessary in order to accomodate the code change at `$scope.delete`, which now depends upon the item's "type" to delete... dashboard items are type "dashboard", so this correction.

See. Legacy_app.js : 174